resources/views/layouts/default.blade.php

```diff
@@ -950,4 +950,5 @@
 
     @livewireScripts
   </body>
+  <script>var FreeScoutW={s:{"cf":["1","2","4","3"],"color":"#0068bd","position":"bl","id":3427502676}};(function(d,e,s){if(d.getElementById("freescout-w"))return;a=d.createElement(e);m=d.getElementsByTagName(e)[0];a.async=1;a.id="freescout-w";a.src=s;m.parentNode.insertBefore(a, m)})(document,"script","https://demo-canidesk.canaris.in/modules/enduserportal/js/widget.js?v=7516");</script>
```

Comment:
@rajeshcanaris Please make the host part of the script url dynamic, so that we do not need to make changes when this is deployed in production. It should use the value of `config('app.url')` .